spec/views/medicaid/health_pregnancy/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+describe "medicaid/health_pregnancy/edit.html.erb" do
+  before do
+    controller.singleton_class.class_eval do
+      def current_path
+        "/steps/medicaid/health-pregnancy"
+      end
+
+      helper_method :current_path
+    end
+  end
+
+  context "2 members - primary member is male" do
+    it "asks if the only *other* female is pregnant" do
+      app = create(:medicaid_application, anyone_new_mom: true)
+      create(:member, :male, first_name: "Joel", benefit_application: app)
+      create(:member, :female, first_name: "Sara", benefit_application: app)
+      pregnancy_step = Medicaid::HealthPregnancy.new(
+        anyone_new_mom: true,
+        members: app.members,
+      )
+
+      assign(:step, pregnancy_step)
+
+      render
+
+      expect(rendered).to include("Has Sara been pregnant recently?")
```

Comment:
is it problematic that we use the word `recently` rather than `in the last 3 months` ? Seems pretty vague

Reply:
Who's got final word on copy? @rachelcope - what do you think?